packages/typography/src/Typography.test.js

```diff
@@ -88,16 +76,3 @@ describe("Typography", () => {
     });
   });
 });
-
-[H1, H2, H3, Text, Body, Bold, Caption, Disabled, Sub1, Sub2].forEach(
```

Comment:
We're not testing these individually? I thought people would end up using the named exports rather than `Typography` directly.